distributed/scheduler.py

```diff
@@ -2126,6 +2126,18 @@ def decide_worker(self, ts: TaskState) -> WorkerState:
             n_workers: Py_ssize_t = len(worker_pool_dv)
             if n_workers < 20:  # smart but linear in small case
                 ws = min(worker_pool.values(), key=operator.attrgetter("occupancy"))
+                if ws.occupancy == 0:  # special case to use round-robin
```

Comment:
Would suggest using the `_` attributes where possible. These are typed, which can have a performance impact (particularly when the scheduler is Cythonized)

```suggestion
                if ws._occupancy == 0:  # special case to use round-robin
```